frontend/app/scripts/directives/editableLabelDirective.js

```diff
@@ -0,0 +1,10 @@
+angular.module('frontendApp').directive("editableLabel", function () {
```

Comment:
Please, add an unique preffix for the directive name. Guys use preffix 'trn-';
There is article with examples https://medium.com/@cironunesdev/angularjs-how-to-name-directives-118ac44b81d4 .